source/app.d

```diff
@@ -147,8 +147,9 @@ auto matchIssueRefs(string message)
 {
     static auto matchToRefs(M)(M m)
     {
+        enum splitRE = regex(`[^\d]+`); // ctRegex throws a weird error in unittest compilation
         auto closed = !m.captures[1].empty;
-        return m.captures[5].stripRight.splitter(ctRegex!`[^\d]+`)
+        return m.captures[5].stripRight.splitter(splitRE)
```

Comment:
With `ctRegex` and the `unittest` below, I was getting the following error:
However I couldn't reproduce this in an isolated case :/


```d
m.captures[5] = ["16319", ""];
```

```d
core.exception.AssertError@/usr/include/dlang/dmd/std/regex/package.d(563): wrong match: 9..4
----------------
??:? _d_assert_msg [0xa47df5]
/usr/include/dlang/dmd/std/regex/package.d:564 std.regex.Captures!(immutable(char)[], ulong).Captures.opIndex!().opIndexinout(pure nothrow @trusted inout(immutable(char)[]) function(ulong)) [0x878691]
source/app.d:155 @safe app.IssueRef[] app.matchIssueRefs(immutable(char)[]).matchToRefs!(std.regex.Captures!(immutable(char)[], ulong).Captures).matchToRefs(std.regex.Captures!(immutable(char)[], ulong).Captures) [0x8772ec]
/usr/include/dlang/dmd/std/algorithm/iteration.d:593 @property @safe app.IssueRef[] std.algorithm.iteration.__T9MapResultS383app14matchIssueRefsFAyaZ11matchToRefsTS3std5regex108__T10RegexMatchTAyaS853std5regex8internal12backtracking29__T19BacktrackingMatcherVbi1Z19BacktrackingMatcherZ10RegexMatchZ.MapResult.front() [0x87e89d]
/usr/include/dlang/dmd/std/algorithm/iteration.d:2376 ref @safe std.algorithm.iteration.joiner!(std.algorithm.iteration.__T9MapResultS383app14matchIssueRefsFAyaZ11matchToRefsTS3std5regex108__T10RegexMatchTAyaS853std5regex8internal12backtracking29__T19BacktrackingMatcherVbi1Z19BacktrackingMatcherZ10RegexMatchZ.MapResult).joiner(std.algorithm.iteration.__T9MapResultS383app14matchIssueRefsFAyaZ11matchToRefsTS3std5regex108__T10RegexMatchTAyaS853std5regex8internal12backtracking29__T19BacktrackingMatcherVbi1Z19BacktrackingMatcherZ10RegexMatchZ.MapResult).Result std.algorithm.iteration.joiner!(std.algorithm.iteration.__T9MapResultS383app14matchIssueRefsFAyaZ11matchToRefsTS3std5regex108__T10RegexMatchTAyaS853std5regex8internal12backtracking29__T19BacktrackingMatcherVbi1Z19BacktrackingMatcherZ10RegexMatchZ.MapResult).joiner(std.algorithm.iteration.__T9MapResultS383app14matchIssueRefsFAyaZ11matchToRefsTS3std5regex108__T10RegexMatchTAyaS853std5regex8internal12backtracking29__T19BacktrackingMatcherVbi1Z19BacktrackingMatcherZ10RegexMatchZ.MapResult).Result.__ctor(std.algorithm.iteration.__T9MapResultS383app14matchIssueRefsFAyaZ11matchToRefsTS3std5regex108__T10RegexMatchTAyaS853std5regex8internal12backtracking29__T19BacktrackingMatcherVbi1Z19BacktrackingMatcherZ10RegexMatchZ.MapResult) [0x87f3ea]
/usr/include/dlang/dmd/std/algorithm/iteration.d:2419 @safe std.algorithm.iteration.joiner!(std.algorithm.iteration.__T9MapResultS383app14matchIssueRefsFAyaZ11matchToRefsTS3std5regex108__T10RegexMatchTAyaS853std5regex8internal12backtracking29__T19BacktrackingMatcherVbi1Z19BacktrackingMatcherZ10RegexMatchZ.MapResult).joiner(std.algorithm.iteration.__T9MapResultS383app14matchIssueRefsFAyaZ11matchToRefsTS3std5regex108__T10RegexMatchTAyaS853std5regex8internal12backtracking29__T19BacktrackingMatcherVbi1Z19BacktrackingMatcherZ10RegexMatchZ.MapResult).Result std.algorithm.iteration.joiner!(std.algorithm.iteration.__T9MapResultS383app14matchIssueRefsFAyaZ11matchToRefsTS3std5regex108__T10RegexMatchTAyaS853std5regex8internal12backtracking29__T19BacktrackingMatcherVbi1Z19BacktrackingMatcherZ10RegexMatchZ.MapResult).joiner(std.algorithm.iteration.__T9MapResultS383app14matchIssueRefsFAyaZ11matchToRefsTS3std5regex108__T10RegexMatchTAyaS853std5regex8internal12backtracking29__T19BacktrackingMatcherVbi1Z19BacktrackingMatcherZ10RegexMatchZ.MapResult) [0x87f2ed]
source/app.d:164 @safe std.algorithm.iteration.joiner!(std.algorithm.iteration.__T9MapResultS383app14matchIssueRefsFAyaZ11matchToRefsTS3std5regex108__T10RegexMatchTAyaS853std5regex8internal12backtracking29__T19BacktrackingMatcherVbi1Z19BacktrackingMatcherZ10RegexMatchZ.MapResult).joiner(std.algorithm.iteration.__T9MapResultS383app14matchIssueRefsFAyaZ11matchToRefsTS3std5regex108__T10RegexMatchTAyaS853std5regex8internal12backtracking29__T19BacktrackingMatcherVbi1Z19BacktrackingMatcherZ10RegexMatchZ.MapResult).Result app.matchIssueRefs(immutable(char)[]) [0x81e169]
source/app.d:169 void app.__unittestL167_8() [0x81e1c7]
??:? void app.__modtest() [0x8ba6c0]
??:? int core.runtime.runModuleUnitTests().__foreachbody2(object.ModuleInfo*) [0xaa6788]
??:? int object.ModuleInfo.opApply(scope int delegate(object.ModuleInfo*)).__lambda2(immutable(object.ModuleInfo*)) [0xa469eb]
??:? int rt.minfo.moduleinfos_apply(scope int delegate(immutable(object.ModuleInfo*))).__foreachbody2(ref rt.sections_elf_shared.DSO) [0xa534c6]
??:? int rt.sections_elf_shared.DSO.opApply(scope int delegate(ref rt.sections_elf_shared.DSO)) [0xa53a0d]
??:? int rt.minfo.moduleinfos_apply(scope int delegate(immutable(object.ModuleInfo*))) [0xa53457]
??:? int object.ModuleInfo.opApply(scope int delegate(object.ModuleInfo*)) [0xa469c7]
??:? runModuleUnitTests [0xaa667a]
??:? void rt.dmain2._d_run_main(int, char**, extern (C) int function(char[][])*).runAll() [0xa4e5aa]
??:? void rt.dmain2._d_run_main(int, char**, extern (C) int function(char[][])*).tryExec(scope void delegate()) [0xa4e54c]
??:? _d_run_main [0xa4e4bd]
??:? main [0x81b677]
??:? __libc_start_main [0x15bd5290]
```